Resources/Prototypes/Entities/Objects/Misc/identification_cards.yml

```
@@ -650,7 +650,7 @@
     - SyndicateAgent
 
 - type: entity
-  parent: IDCardStandard
+  parent: [ IDCardStandard, BaseSyndicateContraband ]
```

```suggestion
  parent: [ IDCardStandard, BaseMajorContraband ]
```

Pirate gear isn't Syndicate contra, and shouldn't be marked as such. If you want it to be contra make it either minor/major.

---

I believe Major here works. I see Syndicate as any **corporation/entity that goes against NanoTrasen**. It's weird that something maybe like a gun or even the ninja sword is only considered Major when those are directly against NT the same way a syndicate soap is. 

Even with that in mind, Major is a good suggestion and I'll change it to that now.